docs/index.html

```diff
@@ -0,0 +1,102 @@
+            <a class="nav-link" href="">Vision</a>
+            <a class="nav-link" href="">Features</a>
+            <a class="nav-link" href="">About</a>
+            <a class="nav-link" href="">Contact</a>
+        </nav>
+
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </header>
+</div>
+
+<div class="container">
+
+    <section class="blurb-section">
+        <div class="blurb-text-div">
+            <h1 class="blurb-text">
+                Innovation<br/>
+                On<br/>
+                Demand<br/>
+            </h1>
+            <div class="blurb-button-div">
+                <button class="blurb-button button">Get Started</button>
+            </div>
+        </div>
+        <div class="blurb-img-div">
+            <img class="blurb-img" src="img/header-img.png" alt="Image of a code snippet.">
+        </div>
+    </section>
+
+    <section class="section section1 main-text">
+        <section class="features section-content2by">
+            <h2>Features</h2>
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+
+        <section class="about section-content2by">
+            <h2>About</h2>
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+    </section>
+
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+    <section class="section section2 main-text">
+        <section class="section-content3by services">
+            <h2>Services</h2>
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+        <section class="section-content3by products">
+            <h2>Product</h2>
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+        <section class="section-content3by vision">
+            <h2>Vision</h2>
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </section>
+    </section>
+    <footer>
+        <h2>Contact</h2>
+        <address>
+            123 Way 456 Street<br/>
+            Somewhere, USA
+        </address>
+        <div class="phone">1 (888) 888-8888</div>
```

Comment:
```suggestion
              <a href="tel:18888888888">1 (888) 888-8888</a>
```
Putting this into an anchor tag and using 'tel:PHONE-NUMBER' will make this more semantic. Your user will know that this is a phone number, overall allowing for a better UX